python/ray/tests/test_multinode_failures.py

```diff
@@ -83,10 +85,16 @@ def _test_component_failed(cluster, component_type):
     # Submit many tasks with many dependencies.
     @ray.remote
     def f(x):
+        if RAY_FORCE_DIRECT:
+            # The test takes too long with the raylet codepath.
+            time.sleep(0.01)
         return x
 
     @ray.remote
     def g(*xs):
+        if RAY_FORCE_DIRECT:
+            # The test takes too long with the raylet codepath.
```

Comment:
The non-direct call version of this test is already really slow, so didn't want to add any more time to it. The sleep is to make sure that some tasks actually fail mid-execution (otherwise they might all finish too quickly).